packages/gatsby-react-router-scroll/src/ScrollContainer.js

```diff
@@ -78,7 +72,12 @@ class ScrollContainer extends React.Component {
   }
 }
 
-ScrollContainer.propTypes = propTypes
-ScrollContainer.contextTypes = contextTypes
+const ScrollContainerConsumer = props => (
+  <ScrollBehaviorContext.Consumer>
+    {context => <ScrollContainer {...props} context={context} />}
+  </ScrollBehaviorContext.Consumer>
+)
```

Comment:
Shouldn't need this wrapper. You can attach to a single context with the static:

https://reactjs.org/docs/context.html#classcontexttype

Reply:
Actually, never mind. That wasn't added until 16.6 which would not match our compatibility table for react versions.

Reply:
Really neat, however available from 16.6.0 onwards. Isn't that going to be an issue?